spec/order_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'order'
+
+describe Order do
+
+	let(:dish) {double :dish}
+	
+	context 'selecting a dish' do
+		it { is_expected.to respond_to(:select_dish) }
+
+		it 'selects dishes from menu' do
+	    expect(subject.select_dish('salmon sashimi', 2)).to eq(2)
+	  end
+
+	  it 'cannot select dishes that are not on the menu' do
+	  	expect{subject.select_dish('fois gras', 2)}.to raise_error "We don't serve that item, sorry!"
+	  end
+	end
+
+	context 'calculating and checking totals' do
+		it { is_expected.to respond_to(:total_cost) }
+
+		it 'should return total cost of order' do
+			subject.select_dish('salmon sashimi', 2)
+			expect(subject.total_cost).to eq("Total cost of yout meal is £#{'%.2f' % 8.00}")
```

Comment:
Not sure about the naming of #total_cost, as it returns a string rather than a price figure.